src/main/java/com/thebois/ColonyManagement.java

```diff
@@ -111,11 +120,19 @@ private void setUpUserInterfaceSkin() {
     }
 
     private void createModels() {
-        world = new World(WORLD_SIZE, PAWN_COUNT);
+        world = new World(WORLD_SIZE);
+        final Collection<IBeing> pawns = new ArrayList<>();
+        final Iterable<Position> vacantPositions = world.findEmptyPositions(PAWN_POSITIONS);
+        final Random random = new Random();
+        final IPathFinder pathFinder = new AstarPathFinder(world);
+        for (final Position vacantPosition : vacantPositions) {
+            pawns.add(new Pawn(vacantPosition, vacantPosition, random, pathFinder));
+        }
```

Comment:
Should be broken out into `createPawns`.

Reply:
And moved into Colony?

Reply:
👍

Reply:
> 
> 
> And moved into Colony?

If we move it to `Colony `then it once again depends on `World`, sort of the point refactor to avoid that.

Reply:
You could close this conversation since we're having the same discussion in the conversation above.